### ISSUE TITLE
GNOME/Wayland fails to start

### ISSUE THREAD

**Issue by rossburton** [clearlinux/distribution#1937]
Release 32990.

If I try to start a GNOME on Wayland session, it spends 20 seconds thrashing the logs with:

```
...
gnome-shell[578]: failed to bind to /tmp/.X11-unix/X4840171: No such file or directory
gnome-shell[578]: failed to bind to /tmp/.X11-unix/X4840172: No such file or directory
gnome-shell[578]: failed to bind to /tmp/.X11-unix/X4840173: No such file or directory
...
```

Eventually systemd kill the shell and it falls back to Xorg, which works.

**Comment by rossburton** [comment]
Initially blaming @mrkz from the GNOME 3.36 upgrade. :)

**Comment by mrkz** [comment]
I had this behavior when I started the GNOME migration and was because of an outdated component (`pulseaudio-12.2` < `13`) as part of the upgrade. However, this was in an insolated mix and it got resolved previous to shipping the updated content to the OS.

The issue was due to an outdated (and incompatible) package, just like the issue in https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1739113.html
Although they were not the same issue, its root-cause was similar.

After I updated the package, I saw the issue to go away (and then proceeded with pushing the content to an update).

I'm not able to reproduce this issue on my end fwiw.

A verbose output from gdm start would he helpful
